venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,362 @@
+                },
+            "track": {
+                "description": "Select the track that best fits your submission.",
+                "values-checkbox": [
+                "Ethics and NLP",
+                "Linguistic theories, Cognitive Modeling and Psycholinguistics",
+                "Machine Learning for NLP",
+                "Phonology, Morphology and Word Segmentation",
+                "Resources and Evaluation",
+                "Semantics: Lexical",
+                "Semantics: Sentence-level Semantics, Textual Inference and Other areas",
+                "Syntax: Tagging, Chunking and Parsing",
+                "Information Extraction",
+                "Computational Social Science and Cultural Analytics",
+                "Information Retrieval and Text Mining",
+                "Interpretability and Analysis of Models for NLP",
+                "Machine Translation and Multilinguality",
+                "NLP Applications",
+                "Question Answering",
+                "Dialogue and Interactive Systems",
+                "Discourse and Pragmatics",
+                "Generation",
+                "Language Grounding to Vision, Robotics and Beyond",
+                "Sentiment Analysis, Stylistic Analysis, and Argument Mining",
+                "Speech and Multimodality",
+                "Summarization",
+                "Special Theme"
+                ],
+                "required": True,
+                "order": 12
+                },
+            "comment": {
+                "description": "Comment to Senior Area Chairs (500 words)",
+                "order": 13,
+                "value-regex": "[\\S\\s]{0,500}",
+                "required": False
+                }
+        }
+    }
+)
+client.post_invitation(commitment)
+
+# Creating SAC groups from Track Name 
+f = open('SAC_groups.csv')
+SAC_name_dictionary = {}
+
+file = csv.reader(f)
+header = []
+rows = []
+header = next(file)
+for row in file:
+    rows.append(row)
+f.close()
+for row in rows:
+    SAC_name_dictionary[row[1]] = row[0]
+
+
+
+# Retrieve all commitment submissions, ACL submissions, and ACL blind submissions  
+notes = openreview.tools.iterget_notes(client,invitation='aclweb.org/ACL/2022/Conference/-/Commitment_Submission')
+submissions = openreview.tools.iterget_notes(client,invitation='aclweb.org/ACL/2022/Conference/-/Submission')
+blind_submissions = {note.forum: note for note in list(openreview.tools.iterget_notes(client, invitation = 'aclweb.org/ACL/2022/Conference/-/Blind_Submission'))}
+
+# Save all submissions in a dictionary by paper_link
+submission_dict = {submission.content['paper_link'].split('=')[1]:submission for submission in submissions}
+
+
+# Iterate through all commitment submissions  
+for note in tqdm(notes):
+    # For each, retrieve old paper from link
+    forum = note.content['paper_link'].split('=')[1]
+    if forum not in submission_dict:
+        original_arr_sub_id = client.get_note(forum).original
```

Comment:
I think get_note throws an exception if the note doesn't exist or use tools.get_note